assets/images/checked.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
```

Comment:
you could minify this SVG with svgo

Reply:
I didn't know that feature I will take a look thank Bryan